README.md

```diff
@@ -56,61 +76,57 @@ A number of default values are used when installing Ansible Tower on the target
+The `bootstrap.yml` playbook will run the Prerequisite, Integreatly, Cluster Create and Cluster Teardown bootstrap playbooks in succession. These individual playbooks can be run independently if required, with instructions on how to do so in the following sections. The playbook requires the target tower environment to be specified.
+
+* `tower_environment`: The Ansible Tower environment (dev/test/qe etc.)
 
 ```bash
-ansible-playbook -i inventories/hosts playbooks/bootstrap.yml -e tower_host=<tower-host> -e tower_username=<tower-username> -e tower_password=<tower-password> -e tower_environment=<tower-environment>
+ansible-playbook -i <path-to-local-credentials-project>/inventories/hosts playbooks/bootstrap.yml -e tower_host=<tower-host> -e tower_username=<tower-username> -e tower_password=<tower-password> -e tower_environment=<tower-environment> --ask-vault-pass
 ```
 
-#### Openshift Dedicated
+#### 4.1.1 Openshift Dedicated
 
 If you also wish to bootstrap the tower instance with the OSD integreatly install workflow you need to run this play after running the bootstrap.yml play.
 
-```
+```bash
 ansible-playbook -i inventories/hosts playbooks/bootstrap_osd_integreatly_install.yml -e tower_environment=<tower-environment>
 ```
 
 This will create the resources necessary to use the *Integreatly_Bootstrap_and_install_[OSD]* workflow which will install Integreatly on a targeted OSD cluster.
 
-### Prerequisite Bootstrapping
-
-Prior to running any jobs stored in this repository, the target Ansible tower instance must first be bootstrapped with some generic resources.
+### 4.2 Prerequisite Bootstrapping
 
-The following bootstrap playbook requires a number of configs to be passed in, namely:
+Prior to running any jobs stored in this repository, the target Ansible tower instance must first be bootstrapped with some generic resources. The playbook requires the target tower environment to be specified. The playbook requires the target tower environment to be specified.
 
-* `tower_host`: The hostname of the target Ansible Tower instance
-* `tower_username`: The username to login with when integrating with Tower (requires admin permissions)
-* `tower_password`: The password of the tower_user specified
-* `tower_verify_ssl`: (Optional) Determines whether to verify the SSL certs installed on the Tower instance. By default this is set to `false`
 * `tower_environment`: The Ansible Tower environment (dev/test/qe)
 
 ```bash
-ansible-playbook -i inventories/hosts playbooks/bootstrap_tower.yml -e tower_host=<tower-host> -e tower_username=<tower-username> -e tower_password=<tower-password> -e tower_environment=<tower-environment>
+ansible-playbook -i <path-to-local-credentials-project>/inventories/hosts playbooks/bootstrap_tower.yml -e tower_host=<tower-host> -e tower_username=<tower-username> -e tower_password=<tower-password> -e tower_environment=<tower-environment>
```

Comment:
All you need to include here is tower_environment, the creds repo will handle the rest